### PR TITLE
getting custom validation from octobermodel

### PR DESCRIPTION
**Pull request by vardash0131**
to get custom validation for example dimensions:width=728,height=90

### CODE REVIEW

File: traits/ComponentUtils.php [hunk 1]
```diff
@@ -124,10 +124,14 @@ public function onUpload()
             if ($fileTypes = $this->processFileTypes()) {
                 $validationRules[] = 'extensions:'.$fileTypes;
             }
+            //custom validations
+            if (isset($this->model->rules[$this->attribute]))
+                $validationRules[] = $this->model->rules[$this->attribute];
```
Comment (LukeTowers) [inline traits/ComponentUtils.php]:
```suggestion

            // Support model validation rules
            if (!empty($this->model->rules[$this->attribute])) {
                $validationRules[] = $this->model->rules[$this->attribute];
            }
```

File: traits/ComponentUtils.php [hunk 2]
```diff
@@ -124,10 +124,14 @@ public function onUpload()
...
 
             $validation = Validator::make(
                 ['file_data' => $uploadedFile],
-                ['file_data' => $validationRules]
+                ['file_data' => $validationRules],
+                $this->model->customMessages?$this->model->customMessages:[]
```
Comment (LukeTowers) [inline traits/ComponentUtils.php]:
```suggestion
                $this->model->customMessages ? $this->model->customMessages : []
```

**Comment by vardash0131** [comment]
nice

**Comment by LukeTowers** [comment]
I'll wait for @daftspunk to review and merge this, but looks good to me.

**Pull request merged**